elastic/apm-server
# Map RUM PerformanceResourceTiming fields

## Motivation/summary

Add field mappings for:

 - http.response.transfer_size
 - http.response.encoded_body_size
 - http.response.decoded_body_size

The fields are mapped with `index: false`, to minimise storage overhead. They can still be used in searches, just a bit slower.

The intake schema for these fields has been changed from float64 to int, as these are originally integer values.
See https://w3c.github.io/resource-timing/#sec-performanceresourcetiming

testdata, and hence approvals, have been updated to change the mock values from floats to integers.

## Checklist


## How to test these changes

1. Send some RUM page-load transactions and HTTP spans to APM Server
2. Ensure the fields can be searched and aggregated

## Related issues

None
